srcpkgs/sddm/template

```diff
@@ -30,6 +30,7 @@ conf_files="
 make_dirs="/var/lib/sddm 0755 sddm sddm"
 
 post_install() {
-	sed -i 's,systemd,ck_connector,g' ${DESTDIR}/etc/pam.d/sddm-greeter
+	vsed -i 's,systemd,ck_connector,g' ${DESTDIR}/etc/pam.d/sddm-greeter
```

Comment:
This check seems to be not needed anymore. Should I delete it?
```
=> WARNING: sddm-0.18.1_3: vsed: regex "s,systemd,ck_connector,g" didn't change file "/destdir//sddm-0.18.1/etc/pam.d/sddm-greeter"
```

Reply:
What does the file look like? A systemd Pam module should be replaced by the elogind one, perhaps. Replacing by the console kit one won't work anymore.

Reply:
In `/etc/pam.d/sddm-greeter` is no entry with systemd but ```session         optional pam_elogind.so``` on my installed sddm.

Reply:
obsolete since https://github.com/sddm/sddm/commit/96e50a76db406ab6a225b988eaaad7b62eadb3a4
that is exactly why vsed is useful

Reply:
Removed